Library/Formula/bitlbee.rb

```diff
@@ -88,4 +88,9 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    output = shell_output("#{sbin}/bitlbee -V", 1)
+    assert_match version.to_s, output
```

Comment:
Oh, sorry, missed the arg. Yeh, just use the `shell_output` and ignore the result, thanks!